### PR TITLE
PR 1: Demo Changes

### PR DESCRIPTION
**Pull request by farbars**
My very first pull request 🎉

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          React Portfolio Wonderful
+          React Portfolio Fantastic
```
Comment (CanonMinds) [inline src/App.js:10]: Sample comment. Cheers! :wine_glass: 

This can be added with :tada: 

Thanks sir @farbars

**Review by CanonMinds — COMMENTED** [review]
Sir @farbars  kindly see comments on your PR. Send me a ping once complete. :dancers:

File: src/App.js (hunk 1 above)
Comment (farbars) [reply src/App.js:10]: Roger that

File: src/App.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          React Portfolio Wonderful
+          React Portfolio Fantastic 🎉
```
Comment (CanonMinds) [inline src/App.js:10]: LGTM. :vulcan_salute: Heap thanks.

Feel free to merge ASAP. Thanks @farbars

**Review by CanonMinds — APPROVED** [review]
Congratulations on your first PR! :heart:

**Pull request merged**